k8s/Makefile

```diff
@@ -5,6 +5,10 @@ export AWS_RESOURCE_STACK=dev-portal-${TARGET_ENVIRONMENT}
 export AWS_REGION ?= us-west-2
 export K8S_CLUSTER_SHORT_NAME ?= eks-oregon
 
+export AWS_BUCKET_NAME = ${AWS_BUCKET_NAME}
+export AWS_ACCESS_KEY_ID = ${AWS_ACCESS_KEY_ID}
+export AWS_SECRET_ACCESS_KEY = ${AWS_SECRET_ACCESS_KEY}
+
```

Comment:
@limed Can you cast some laser eyes over this and the Docker changes, please? Am pretty sure the vars mentioned above all exist already in k8s, yep?

Reply:
Yes this variables already exists in the k8s manifest so you shouldn't need to worry about doing this here

Reply:
Yes these already exists in k8s you don't even need this here I believe

Reply:
OK will delete them. Cheers!